src/Components/__stories__/Input.story.tsx

```diff
@@ -49,6 +50,30 @@ storiesOf("Components/Input", module)
       <TextArea placeholder="Your Message" disabled />
     </div>
   ))
+  .add("Check Boxes", () => (
+    <div>
+      <div style={{ padding: 10 }}>
+        <Checkbox onChange={event => console.log("I'll remember you!", event)}>
```

Comment:
I left this `console.log` intentionally but the CI doesn't like it...

Reply:
`// tslint:disable-next-line`?

Reply:
I ended up getting rid of it since tslint doesn't recognize `tslint:disable` comments in JSX.